compiler/rustc_session/src/config/cfg.rs

```diff
@@ -463,6 +467,11 @@ impl CheckCfg {
             ins!(sym, no_values).extend(atomic_values);
         }
 
+        ins!(sym::target_has_reliable_f16, no_values);
+        ins!(sym::target_has_reliable_f16_math, no_values);
+        ins!(sym::target_has_reliable_f128, no_values);
+        ins!(sym::target_has_reliable_f128_math, no_values);
```

Comment:
If I understood correctly those cfgs are intended to be purely internal. They should therefor not be considered be to well-known (for Rust users).

```suggestion
```

Instead please add the cfgs in the appropriate `Cargo.toml`, like this:
```diff
+    'cfg(target_has_reliable_f16)',
+    'cfg(target_has_reliable_f16_math)',
+    'cfg(target_has_reliable_f128)',
+    'cfg(target_has_reliable_f128_math)',
```
https://github.com/rust-lang/rust/blob/496145b9cc023aef4bb1f16c0964a53d0da36c88/library/std/Cargo.toml#L158-L166
You will also need to re-bless all the check-cfg UI tests.